users/utils/decorators.py

```diff
@@ -23,5 +24,15 @@ def wrapper(self, request, *args, **kwargs):
         user = request.user
         if user.role != CustomUser.EMPLOYEE:
             return Response({'error': 'You do not have permission to access'}, status=status.HTTP_403_FORBIDDEN)
-        return func(self, request, *args, **kwargs)  
+        else: 
+            return func(self, request, *args, **kwargs)  
     return wrapper
+
+def manager_and_employee_required(func):
```

Comment:
are you tested this decorator via postman to check it's working or not?

I think it's not working.
Check the APIs where you use this decorator.